gensim/utils.py

```diff
@@ -20,7 +20,7 @@
 try:
     import cPickle as _pickle
 except ImportError:
-    import pickle as _pickle
+    import _pickle
```

Comment:
Where does this `_pickle` come from? Travis fails on Python3, I think it's related.